packages/grid/data-grid/src/tests/state.DataGrid.test.tsx

```diff
@@ -9,6 +9,7 @@ describe('<DataGrid /> - State', () => {
   const render = createClientRenderStrictMode();
 
   const baselineProps = {
+    autoHeight: /jsdom/.test(window.navigator.userAgent),
```

Comment:
different pattern here.

Reply:
A shortcut. I can create a NO_LATOUT variable if you prefer